presto-hive/src/main/java/io/prestosql/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -108,12 +108,12 @@
         implements HiveSplitLoader
 {
     private static final Iterable<Pattern> BUCKET_PATTERNS = ImmutableList.of(
+            // legacy Presto naming pattern (current version matches Hive)
```

Comment:
Is there any reason to change the ordering of the patterns? It's best not to mix formatting changes with functional changes, as that makes the diff harder to read.